test/src/main/java/org/apache/accumulo/test/functional/FunctionalTestUtils.java

```diff
@@ -81,6 +83,19 @@ public static int countRFiles(AccumuloClient c, String tableName) throws Excepti
     }
   }
 
+  public static List<String> getRFileNames(AccumuloClient c, String tableName) throws Exception {
```

Comment:
This name seems off.  Seems like its returning paths or URIs and not names.  Also could possible use Ample for this, but not quite sure. It may not be easy to use Ample unless you have a ServerContext.